content/sensu-go/5.0/reference/rbac.md

```diff
@@ -88,134 +235,409 @@ username     |
+required     | true 
+type         | Array
+example      | {{< highlight shell >}}"verbs": ["get", "list"]{{< /highlight >}}
 
-type         | 
+resources         | 
 -------------|------ 
-description  | The type of resource the rule has permission to access. 
+description  | The type of resource that the rule has permission to access. Roles can only access [namespaced resource types][17] while cluster roles can access namespaced and [cluster-wide resource types][18]. See [resource types][4] for available types.
 required     | true 
-type         | String
-example      | {{< highlight shell >}}"type": "*"{{</highlight>}}{{< highlight shell >}}"type": "checks"{{</highlight>}}{{< highlight shell >}}"type": "environments"{{</highlight>}}
+type         | Array
+example      | {{< highlight shell >}}"resources": ["checks"]{{< /highlight >}}
 
-organization | 
+resourceNames    | 
 -------------|------ 
-description  | The name of the organization the rule belongs to. 
+description  | Specific resource names that the rule has permission to access. Resource name permissions are only available for `get`, `delete`, and `update` verbs.
+required     | false
+type         | Array
+example      | {{< highlight shell >}}"resourceNames": ["check-cpu"]{{< /highlight >}}
+
+## Role bindings and cluster role bindings
+
+A **role binding** assigns a **role** or **cluster role** to a user or set of users.
+A **cluster role binding** assigns a **cluster role** to a user or set of users.
+Roles bindings apply roles within a namespace while cluster role bindings apply across namespaces and resource types.
+
+To create and manage role bindings within a namespace, [create a role][25] with `rolebindings` permissions within that namespace, and log in by [configuring sensuctl][26].
+
+### Cluster role bindings
+
+Cluster roles bindings can assign a cluster role to users and groups.
+Cluster role bindings use the same [specification][30] as role bindings and can be managed using the same sensuctl commands with `cluster-role-binding` substituted for `role-binding`.
+
+To create and manage cluster role bindings, [configure sensuctl][26] as the [default `admin` user][20] or [create a cluster role][28] with permissions for `clusterrolebindings`.
+
+### Viewing role bindings
+
+You can use [sensuctl][2] to see a list of role bindings within Sensu:
+
+{{< highlight shell >}}
+sensuctl role-binding list
+{{< /highlight >}}
+
+To see the details for a specific role binding:
+
+{{< highlight shell >}}
+sensuctl role-binding info [BINDING-NAME]
+{{< /highlight >}}
+
+To see a list of cluster role bindings:
+
+{{< highlight shell >}}
+sensuctl cluster-role-binding list
+{{< /highlight >}}
+
+### Creating a role binding
+
+You can use [sensuctl][2] to see a create a role binding:
+
+{{< highlight shell >}}
+sensuctl role-binding create [NAME] --role=NAME [--user=username] [--group=groupname]
```

Comment:
We should probably indicate that a role binding and either take a role or a cluster-role (e.g. `--cluster-role=NAME|--role=NAME`)